ocs_ci/helpers/cephx_keyrotation_helper.py

```
@@ -0,0 +1,4389 @@
+        )
+        try:
+            self._get_storagecluster_ocp().patch(
+                params=json.dumps(patch_ops), format_type="json"
+            )
+        except CommandFailed as exc:
+            err = str(exc)
+            if expected_error not in err:
+                raise UnexpectedBehaviour(
+                    "Expected StorageCluster validation error containing "
+                    f"'{expected_error}', got: {err}"
+                )
```

Ruff (0.16.0)

[warning] Within an `except` clause, raise exceptions with `raise ... from err` or `raise ... from None` to distinguish them from errors in exception handling

(B904)